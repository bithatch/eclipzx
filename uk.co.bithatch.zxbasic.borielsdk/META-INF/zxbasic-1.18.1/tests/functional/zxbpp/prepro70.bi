DIM b
#ifndef __INCLUDED__
#define __INCLUDED__  'Avoid double inclusion

#include ".\prepro70.bi" 'Windows backlash style

DIM a

#endif
