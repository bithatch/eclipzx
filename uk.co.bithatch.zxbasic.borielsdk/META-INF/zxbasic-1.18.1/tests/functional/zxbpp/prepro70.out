#line 1 "prepro70.bi"
DIM b



#line 1 "./prepro70.bi"
DIM b
#line 10 "./prepro70.bi"
#line 6 "prepro70.bi"

DIM a

#line 10 "prepro70.bi"
